imap_processing/cdf/tests/imap_test_variable.yaml

```diff
@@ -15,12 +15,13 @@ test_field_1:
   LABLAXIS: test_labaxis
   UNITS: test_units
   VAR_TYPE: test_variable_type
-  SCALETYP: test_scaletyp
-  MONOTON: test_monoton
   TIME_BASE: 10
   TIME_SCALE: test_time_scale
   REFERENCE_POSITION: test_reference_position
-  NOT_REQUIRED: test_not_required
+  LEAP_SECONDS_INCLUDED: test_not_required
+  DEPEND_1: test_depend_1
```

Comment:
Should include `DEPEND_0` in your test data. I think it's debatable whether we want to require `DEPEND_0` in the attribute definition since it should always be present, but I think most of us have been explicitly defining it, so your tests should reflect that.

Reply:
Yes! For `test_field_1`, DEFAULT_0 is defined in default_attrs. However `test_field_3` does not have `DEPEND_0`, but I sent you a few slack questions about this case specifically.

Reply:
Ok, in light of the discussion we had, disregard this comment